ern-runner-gen-ios/src/hull/ErnRunner/ViewController.m

```diff
@@ -47,20 +49,39 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 - (NSString * _Nonnull)rootComponentName {
     return MainMiniAppName;
 }
-    
+
 - (NSDictionary * _Nullable)properties {
     return nil;
 }
-    
+
+//`finisedCallback` is deprecated, use `finish`
+//finishedCallback, finish and nativeNavigation are optional properties
 - (MiniAppFinishedCallback _Nullable)finishedCallback {
```

Comment:
Can we remove this from the runner, since this is a code that is generated we don't need to keep this here? Perhaps we can add a deprecated note in the Protocol level.